### PR TITLE
Support multi-objective study for `from_optuna`

### PR DESCRIPTION
**Pull request by nzw0301**
Hi, thank you for introducing the Optuna integration by #215!

I suppose the current implementation does not support `study` whose objective function returns multiple objective values. More concretely, the following code:

```python
import optuna
import hiplot as hip

def objective(trial: "optuna.trial.Trial") -> float:
    x = trial.suggest_float("x", -1, 1)
    y = trial.suggest_float("y", -1, 1)    

    return x ** 2, y

study = optuna.create_study(directions=["minimize"]*2)
study.optimize(objective, n_trials=3)

xp = hip.Experiment.from_optuna(study)
```

The error message is as follows.
```bash
RuntimeError                              Traceback (most recent call last)
/var/folders/n3/7_7r1yrx6jsc_0780bvg02lr0000gn/T/ipykernel_82314/3836556673.py in <module>
      2 study.optimize(objective, n_trials=3)
      3 
----> 4 xp = hip.Experiment.from_optuna(study)

~/Documents/hiplot/hiplot/experiment.py in from_optuna(study)
    519         hyper_opt_data = []
    520         for each_trial in study.trials:
--> 521             trial_params = {}
    522             trial_params["value"] = each_trial.value # name = value, as it could be RMSE / accuracy, or any value that the user selects for tuning
    523             trial_params["uid"] = each_trial.number

/opt/homebrew/Caskroom/miniconda/base/envs/optuna39/lib/python3.9/site-packages/optuna/trial/_frozen.py in value(self)
    389         if self._values is not None:
    390             if len(self._values) > 1:
--> 391                 raise RuntimeError(
    392                     "This attribute is not available during multi-objective optimization."
    393                 )

RuntimeError: This attribute is not available during multi-objective optimization.
```

Dependencies:
- `Optuna`: 2.10.0
- `hiplot`: https://github.com/facebookresearch/hiplot/tree/79b3d52a6842d6ba12f0a544e27a444562a486df

---

To clarify it in the documentation, this PR mentions the supported study.

By the way, to access the objective values of either single and multi-objective, we can use `tiral.values` that contains retuned value by the objective function of Optuna as list.

### CODE REVIEW

**Comment by nzw0301** [comment]
Definitely! b063c53

File: hiplot/test_experiment.py [hunk 1]
```diff
@@ -59,6 +59,26 @@ def objective(trial: "optuna.trial.Trial") -> float:
     xp._asdict()
 
 
+def test_from_optuna_multi_objective() -> None:
+
+    def objective(trial: "optuna.trial.Trial") -> float:
```
Comment (danthe3rd) [inline hiplot/test_experiment.py:64]: This should make the CI happy :)
(you can test locally with `mypy --strict --implicit-reexport hiplot`, from the root of the git repo)
```suggestion
    def objective(trial: "optuna.trial.Trial") -> tp.Tuple[float, float]:
```

Comment (nzw0301) [reply hiplot/test_experiment.py]: Indeed! Thank you for your comment. I fixed it locally, so I noticed your suggestion after updating this PR.

**Comment by danthe3rd** [comment]
CI is all green - merging
I'm pushing version `0.1.31` with this fix - should be available within ~1h if everything goes well.
Thanks both of you :)

**Pull request merged**